public_suffix_list.dat

```diff
@@ -13809,4 +13813,4 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
-// ===END PRIVATE DOMAINS===
+// ===END PRIVATE DOMAINS===
```

Comment:
What the ?  

Could have merged had there not been the strangeness on line 13816, but cannot merge this as-is.

There is a trailing space and perhaps a missing linefeed? 

Need that fixed, but really, now that we have iterated so many times, I suggest you could take a fresh start - rebase a branch and add the four important lines.